### PR TITLE
docs(k8s): managed control plane overview

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: containers/kubernetes/reference-content/kubernetes-control-plane-offers.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+These components encompass the control plane's core elements, namely etcd, API server, scheduler, cloud controller, and controller manager.
+
+Scaleway offers a range of Kubernetes control planes designed to cater to a wide range of use-cases and provides you with a maximum of flexibility. Whether you are seeking a mutualized environment or a dedicated control plane, we provide a flexible offer to fit your infrastructure needs, which can be a differentiating advantage from other Kubernetes engine providers. 
+
+## Available control plane offers
+
+| Control plane type /<br />Features | Mutualized<br />(free tier) | Dedicated 4       | Dedicated 8       | Dedicated 16        |
+|--------------------|---------------------------|-------------------|-------------------|--------------------|
+| Memory             | up to 4GB<sup>1</sup>.    | 4GB RAM dedicated | 8GB RAM dedicated | 16GB RAM dedicated |
```
Comment (SamyOubouaziz) [inline containers/kubernetes/reference-content/kubernetes-control-plane-offers.mdx:26]:
```suggestion
| Memory             | up to 4GB<sup>1</sup>    | 4GB RAM dedicated | 8GB RAM dedicated | 16GB RAM dedicated |
```

File: containers/kubernetes/reference-content/kubernetes-control-plane-offers.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  Migrating to a lower tier will have some impact on key features. The allocated RAM resources will be reduced, the maximum number of nodes will be limited, and in the case of a mutualized control plane, the SLA (Service Level Agreement) will be revoked.
+</Message>
+
+If a dedicated cluster is deleted, the commitment is automatically canceled, and there will be no further billing for this dedicated service.
+
+## Changing a control plane offer
+
+During the lifecycle of a Kubernetes cluster, you have the flexibility to transition its control plane to a different type, whether it involves [scaling up or scaling down](/faq/kubernetes/#can-i-change-the-plan-of-my-cluster's-control-plane), in order to align with their specific workloads.
+
+For more information regarding the [choice of control plane offers for clusters](/faq/kubernetes/#can-i-choose-the-type-of-control-plane-for-my-cluster), as well as instructions on how to manage a control plane through the [console](/containers/kubernetes/how-to/edit-cluster/) or [API](https://www.scaleway.com/en/developers/api/kubernetes/), refer to our dedicated documentation.
```
Comment (nerda-codes) [inline containers/kubernetes/reference-content/kubernetes-control-plane-offers.mdx:62]: Isn't there a link missing at "refer to our dedicated documentation"?

Comment (bene2k1) [inline containers/kubernetes/reference-content/kubernetes-control-plane-offers.mdx:62]:
```suggestion
Refer to our dedicated documentation for more information regarding the [choice of control plane offers for clusters](/faq/kubernetes/#can-i-choose-the-type-of-control-plane-for-my-cluster), as well as instructions on how to manage a control plane through the [console](/containers/kubernetes/how-to/edit-cluster/) or [API](https://www.scaleway.com/en/developers/api/kubernetes/).
```

File: containers/kubernetes/reference-content/kubernetes-control-plane-offers.mdx (hunk 1 above)
Comment (SamyOubouaziz) [reply containers/kubernetes/reference-content/kubernetes-control-plane-offers.mdx]: Space bewteen numbers and units except for %

**Review by SamyOubouaziz — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**